tensorflow/contrib/image/python/ops/image_ops.py

```diff
@@ -122,6 +146,39 @@ def angles_to_projective_transforms(angles, image_height, image_width):
       axis=1)
 
 
+def translations_to_projective_transforms(translations):
```

Comment:
We already expose angles_to_projective_transforms so that users can compose transformation matrices, so my plan was to make similar public methods for any transformation.